src/cata_tiles.h

```diff
@@ -385,7 +459,33 @@ class cata_tiles
          */
         bool nv_goggles_activated;
 
-
+        //pixel minimap cache methods
+        SDL_Texture_Ptr create_minimap_cache_texture(int tile_width, int tile_height);
+        void process_minimap_cache_updates();
+        void update_minimap_cache( const tripoint& loc, pixel& pix );
+        void prepare_minimap_cache_for_updates();
+        void clear_unused_minimap_cache();
```

Comment:
Seeing all the functions and data members that have the word "minimap" in them, it might be at the time to move all the minimap related stuff into a separate class, but this is out of the range of this PR.